augur/filter/_run.py

```diff
@@ -15,13 +15,16 @@
 from augur.io.print import print_err
 from augur.io.vcf import is_vcf as filename_is_vcf, write_vcf
 from augur.types import EmptyOutputReportingMethod
-from . import include_exclude_rules
+from . import config, include_exclude_rules
 from .io import cleanup_outputs, read_priority_scores
 from .include_exclude_rules import apply_filters, construct_filters
 from .subsample import PriorityQueue, TooManyGroupsError, calculate_sequences_per_group, create_queues_by_group, get_groups_for_subsampling
 
 
 def run(args):
+    # Set run time configuration values.
+    config.ARGS = args
```

Comment:
Ah, I don't think we should make this change. I think it will increase the tangling/spaghetti-ness of the code over time by introducing dynamic globals and action-at-a-distance. As much as possible, we should be preferring "pure" functions.

Reply:
Nod, sounds good.